linera-execution/src/committee.rs

```diff
@@ -30,11 +31,46 @@ pub struct ValidatorState {
     pub votes: u64,
 }
 
+#[derive(Clone, Debug, Default, Eq, Hash, PartialEq)]
+pub struct Validators(pub BTreeMap<ValidatorName, ValidatorState>);
```

Comment:
Here, we should actually just implement `Serialize` and `Deserialize` for `ValidatorName` following the model of `PublicKey` in `crypto.rs`.